opentelemetry-semantic-conventions/scripts/generate-consts-from-spec.sh

```diff
@@ -20,54 +20,24 @@ git fetch origin "v$SPEC_VERSION"
 git reset --hard FETCH_HEAD
 cd "$CRATE_DIR"
 
-docker run --rm \
```

Comment:
We only need to call Weaver once, as the `weaver.yaml` file specifies what needs to be generated.